### DISCUSSION TITLE
Time Series panel: Multiple y-axes without changing units?

### DISCUSSION THREAD

**Discussion by nabelekt** [grafana/grafana#39619] [Dashboards, Panels & Visualizations]
It seems to me that the only way to assign a particular series to use a separate y-axis from other series in the same Time Series panel is to change the units of that series. Is this the case? In the older Graph panel, I could easily assign any series to use a second y-axis with no other changes needed. I'm looking for that behavior in the Time Series panel.

**Comment by mave007** [comment]
For this case I'm using  an `Overrides` with `Fields with name matching regex` and then I use the `Axis`->`Placement` although it will only have a max of 2 y-axis for left or right options

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
